### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by Isengo1989**
Add Readme file for better communication and problem solving

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+[![Build and deploy](https://github.com/shopware/docs/actions/workflows/deploy-developer-portal.yml/badge.svg)](https://github.com/shopware/docs/actions/workflows/deploy-developer-portal.yml)
+[![Issues](https://img.shields.io/github/issues/shopware/docs)](https://github.com/shopware/docs/issues)
+[![Pull Requests](https://img.shields.io/github/issues-pr/shopware/docs)](https://github.com/shopware/docs/pull-requests)
+[![Website](https://img.shields.io/website?url=https%3A%2F%2Fdeveloper.shopware.com)](https://developer.shopware.com)
+
+## This repository
+
+- provides general **developer** documentation for shopware 6
+- is one of the sources for our [developer portal](https://github.com/shopware/developer-portal)
+- represents the content of https://developer.shopware.com/docs/
```
Comment (elkmod) [inline README.md:12]:
```suggestion
This repository contains general **developer** documentation for Shopware 6 which is served on [developer.shopware.com/docs](https://developer.shopware.com/docs/). The root of our developer portal [developer.shopware.com](https://developer.shopware.com/) is reflected by the [developer portal](https://github.com/shopware/developer-portal) repository.
```

**Review by elkmod — CHANGES_REQUESTED** [review]
Just small adjustments, which will improve the reading flow.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This repository contains general **developer** documentation for Shopware 6 which is served on [developer.shopware.com/docs](https://developer.shopware.com/docs/). The root of our developer portal [developer.shopware.com](https://developer.shopware.com/) is reflected by the [developer portal](https://github.com/shopware/developer-portal) repository.
+
+
+![Developer docs](./assets/developer-docs.png)
+
+
+## Workflows
+
+They are defined in the [`.github/workflows`](./.github/workflows) folder and help the DX team to maintain the documentation. The workflows are triggered by events like `push`, `pull_request`, `schedule` and `workflow_dispatch`. The workflows are used to build, test, and deploy the documentation. If a PR pipeline fails, the PR will be marked as failed, and the PR will not be able to be merged. In this case, check the above-mentioned workflows to see what went wrong and where to fix it.
```
Comment (sushmangupta) [inline README.md:16]:
```suggestion
They are defined in the [`.github/workflows`](./.github/workflows) folder and help the DX team to maintain the documentation. The workflows are triggered by events like `push`, `pull_request`, `schedule` and `workflow_dispatch`. The workflows are used to build, test, and deploy the documentation. If a PR pipeline fails, the PR will be marked as failed, and it won't be possible to merge it. In this case, check the below-mentioned workflows to see what went wrong and where to fix it.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+sort .wordlist.txt -o .wordlist.txt
+```
+
+### Media file formats
+
+All files that are linked in markdown should be placed in the `assets` folder. The `assets` folder is used to store all images, videos, and other files that are linked in the markdown files. For more information, check [Methodize Assets](./resources/guidelines/documentation-guidelines/05-methodize-assets.md).
+
+### Validate external links
+
+Using [lychee](https://github.com/lycheeverse/lychee) to validate external links.
```
Comment (sushmangupta) [inline README.md:41]:
```suggestion
### Validate external links

[Lychee](https://github.com/lycheeverse/lychee) is used to check if external links are working properly or broken. The configuration for lychee can be found in [`validate-external-links.yml`](./.github/workflows/validate-external-links.yml).
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

**Comment by sushmangupta** [comment]
May I know what these tags are meant for : https://github.com/shopware/docs/tags ?

**Pull request merged**